.circleci/config.yml

```diff
@@ -2,15 +2,15 @@ version: 2
 jobs:
    test:
      docker:
-       - image: envoyproxy/envoy-build:52f6880ffbf761c9b809fc3ac208900956ff16b4
+       - image: envoyproxy/envoy-build:61b38528d7e46ced9d749d278ba185332310ca95
```

Comment:
nit: Can you fix the Circle config to use the same ref semantics for the image as is done here: https://github.com/envoyproxy/envoy/blob/master/.circleci/config.yml#L1

Reply:
done